TimeTrackerApp/TimeTrackerApp/Controllers/StatusPageViewController.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  ViewController.swift
+//  TimeTrackerApp
+//
+//  Created by Ensar Batuhan Ünverdi on 14.09.2022.
+//
+
+import UIKit
+
+class StatusPageViewController: UIViewController {
+    @IBOutlet weak var timerLabel: UILabel!
+    @IBOutlet weak var taskNameLabel: UILabel!
+    @IBOutlet weak var moreButton: UIButton!
+    @IBOutlet weak var seeAllButton: UIButton!
+    @IBOutlet weak var tableView: UITableView! {
```

Comment:
eğer dışarıdan kullanmıyorsan private olması daha iyi olur. Open/Closed prensibine uyması için dışarıdan erişilmemesi lazım